cypress-tests/cypress/e2e/PaymentMethodListTest/00000-PaymentMethodListConfigCurrency.cy.js

```diff
@@ -0,0 +1,81 @@
+import apiKeyCreateBody from "../../fixtures/create-api-key-body.json";
+import createConnectorBody from "../../fixtures/create-connector-body.json";
+import getConnectorDetails from "../PaymentMethodListUtils/utils";
+import merchantCreateBody from "../../fixtures/merchant-create-body.json";
+import * as utils from "../PaymentMethodListUtils/utils";
+import {
+  card_credit_enabled,
+  bank_redirect_ideal_enabled,
+  create_payment_body_in_EUR,
+} from "../PaymentMethodListUtils/Common";
+import State from "../../utils/State";
+
+// Testing for scenario:
+// MCA1 -> Stripe configured with ideal = { country = "NL", currency = "EUR" }
+// MCA2 -> Cybersource configured with credit = { currency = "USD" }
+// Payment is done with currency as EUR and no billing address
+// The resultant Payment Method list should only have ideal with stripe
+
+let globalState;
+describe("Account Create flow test", () => {
```

Comment:
Since all 5 tests are related to a single context, I feel like moving them to single file and name it better for more context.

You can do something like:

```js
describe() {
	context() {}
	context() {}
	context() {}
	...
}
```

this way, when tests are run, it will look a lot cleaner as well.